integration-cli/docker_cli_exec_test.go

```diff
@@ -639,3 +617,64 @@ func (s *DockerSuite) TestExecWindowsOpenHandles(c *check.C) {
 		break
 	}
 }
+
+func (s *DockerSuite) TestExecKill(c *check.C) {
```

Comment:
`integration-cli` is deprecated and freezed. Please add this test to `integration`